src/main/java/com/icthh/xm/uaa/UaaApp.java

```diff
@@ -78,49 +76,58 @@ private static void initContexts() {
         MdcUtils.putRid(MdcUtils.getRid() + "::" + superKey.getValue());
     }
 
-    @PreDestroy
```

Comment:
Please do not remove this log. Sometimes it helps to identify exact point when application was shutdown.

Reply:
done